math/mathmore/inc/Math/Polynomial.h

```diff
@@ -116,20 +116,20 @@ class Polynomial : public ParamFunction<IParamGradFunction>,
       equation is very small. In that case it might be more robust to use the numerical method, by calling directly FindNumRoots()
 
    */
-   const std::vector<std::complex <double> > & FindRoots();
+   const std::vector<std::complex <double> > FindRoots() const;
```

Comment:
Now that we return by value, we don't need to impose constness on the user of this function, so we are actually making things easier to use! 🙂 
```suggestion
  std::vector<std::complex <double> > FindRoots() const;
```